riocli/deployment/util.py

```diff
@@ -191,3 +192,36 @@ def print_deployments_for_confirmation(deployments: List[Deployment]):
     data = [[d.name, d.deploymentId, d.phase, d.status] for d in deployments]
 
     tabulate_data(data, headers)
+
+
+def process_deployment_errors(errors: List, no_action: bool = False) -> str:
+    err_fmt = '[{}] {}\nAction: {}'
+    support_action = ('Report the issue together with the relevant'
+                      ' details to the support team')
+
+    action, description = '', ''
+    msgs = []
+    for code in errors:
+        if code in ERRORS:
+            description = ERRORS[code]['description']
+            action = ERRORS[code]['action']
+        elif code.startswith('DEP_E2'):
+            description = 'Internal rapyuta.io error in the components deployed on cloud'
+            action = support_action
+        elif code.startswith('DEP_E3'):
```

Comment:
does this indicate internal rapyuta.io component error for an image pull error, if yes then i think we need to update the description and support action also

Reply:
No, the image pull error is documented in the errors.py file. If a match is not found, then we enter these conditions.
https://github.com/rapyuta-robotics/rapyuta-io-cli/blob/devel/riocli/deployment/errors.py